tailwind.config.js

```diff
@@ -418,10 +418,13 @@ module.exports = {
   },
   variants: {
     extend: {
-      textColor: ['active'],
+      textColor: ['active', 'important'],
       backgroundColor: ['active'],
       contrast: ['hover', 'focus'],
     },
   },
-  plugins: [],
+  plugins: [require('tailwindcss-important')()],
+  experiments: {
+    pluginVariants: true,
+  },
```

Comment:
Please undo changes in this file since `important` is a bad practice, it can and should be avoided

Reply:
I think it is necessary

Reply:
I'm pretty sure important should be avoided

Reply:
Let me know I'm missing something and suggested changes won't resolve a problem